docs/tutorials/control/air-quality-fleet.md

```diff
@@ -797,28 +749,247 @@ You can find all the code in the [GitHub repo for this tutorial](https://github.
+
+- When you provision Antonia's machines, you create them inside a new location called `Antonia's Home` and grant Antonia operator access to the location.
+  This will later allow her to view data from the air sensors at her home.
+- When you provision the machines for RobotsRUs, you create a location called `RobotsRUs` and two sub-locations for `New York Office` and `Oregon Office`.
+  Then you create the machines in the sub-locations and grant RobotsRUs operator access to the `RobotsRUs` machines location.
+
+You, as the organization owner, will be able to manage any necessary configuration changes for all air sensing machines in all locations created within the Pollution Monitoring Made Simple organization.
+
+{{<imgproc class="imgzoom" src="/tutorials/air-quality-fleet/example-org-structure.png" resize="x900" declaredimensions=true alt="Diagram of the Pollution Monitoring Made Simple organization. In it are two locations: Antonia's HOme and Robots R Us. Robots R Us contains two sub-locations, each containing some machines. The Antonia's Home location contains two machines (and no sub-locations)." style="width:800px">}}
+
+For more information, see [Fleet Management](/manage/reference/organize/) and [provisioning](/manage/fleet/provision/setup/).
+
+### Organize your fleet
+
+If you want to follow along, create the following locations:
+
+- `Antonia's Home`
+- `RobotsRUs`
+
+For `RobotsRUs` crate two sublocations:
+
+1. Add a new location called `Oregon Office` using the same **Add location** button.
+1. Then, find the **New parent location** dropdown on the Oregon Office page.
+1. Select **RobotsRUs** and click **Change**.
+
+Repeat to add the New York office: Add a new location called `New York Office`, then change its parent location to **RobotsRUs**.
+
+{{<imgproc class="imgzoom" src="/tutorials/air-quality-fleet/locations-done.png" resize="x900" declaredimensions=true alt="The New York Office fleet page. The left Locations navigation panel lists Antonia's Home and RobotsRUs, with New York Office and Oregon Office nested inside RobotsRUs." style="width:600px"  >}}
+
+## Getting machines ready for third parties
+
+Continuing with our fictitious company, let's assume you want to ship air sensing machines to customers as ready-to-go as possible.
+In other words, you want to provision devices.
+
+Before an air sensing machine leaves your factory, you'd complete the following steps:
+
+1. You'd flash the single-board computer with an operating system
+2. You'd install `viam-agent`
+3. You'd provide a machine configuration template, a fragment.
+
+Once a customer receives your machine, they will:
+
+1. Plug it in and turn it on.
+2. `viam-agent` will start a WiFi network
+3. The customer uses another device to connect to the machine's WiFi network and the user gives the machine the password for their WiFi network.
+4. The machine can now connect to the internet and complete setup based on the fragment it knows about.
```

Comment:
"the fragment it knows about" is not clear and is a bit informal. Consider rewriting.